README.md

```diff
@@ -1,3 +1,26 @@
+# s4ros/scalpel
+
+In courtesy of https://github.com/sleuthkit/scalpel
```

Comment:
This segment should be removed, since this will actually be in the official  `sleuthkit` repository